Extension/package-lock.json

```diff
@@ -2194,7 +2194,8 @@
         "ansi-regex": {
           "version": "2.1.1",
           "bundled": true,
-          "dev": true
+          "dev": true,
+          "optional": true
```

Comment:
Do you know what's going on with this?  A recent change removed the "optional" and now it's back.

Reply:
Not sure yet. That's just what I got when I ran npm install.